lib/api/controllers/authController.js

```diff
@@ -101,7 +100,8 @@ class AuthController extends BaseController {
     return this.kuzzle.repositories.token.expire(request.context.token)
       .then(() => ({acknowledged: true}))
       .catch(err => {
-        const error = new KuzzleInternalError('Error while forcing token expiration.');
+        //const error = new KuzzleInternalError('Error while forcing token expiration.');
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ACommentedCode)